1.8/install/database-restore.md

```diff
@@ -0,0 +1,163 @@
+
+3. Select "Choose File", browse to your backup, and click "Open" on the Windows Prompt.
+
+4. Click "Execute". Allow the SQL to run. If successful, you will be able to scroll down to the bottom of the page and it will look something like this:
+
+[![phppgadmin successful](/assets/images/1.8/database-restore/phppgadmin_3.png)](/assets/images/1.8/database-restore/phppgadmin_3.png)
+
+## Database Restore using SSH
+
+### MySQL
+
+1. After logging into your server via SSH, run the following:
+
+```
+mysql -u YOUR_MYBB_DB_USER -p YOUR_MYBB_DBNAME < /PATH/TO/DUMP.SQL
+```
+
+2. You will be prompted for your password. If successful, your SSH window will look something like this:
+
+[![importssh screen](/assets/images/1.8/database-restore/ssh_mysql_1.png)](/assets/images/1.8/database-restore/ssh_mysql_1.png)
+
+3. To be sure, log back into MySQL by running the following (When prompted, enter password):
+
+```
+mysql -u YOUR_MYBB_DB_USER -p
+```
+
+4. Now run the following to select your database and show existing tables:
+
+```
+USE database_name_here; SHOW TABLES;
+```
+
+5. If successful, your SSH window will look something like this:
+
+[![ssh mysqltables](/assets/images/1.8/database-restore/ssh_mysql_2.png)](/assets/images/1.8/database-restore/ssh_mysql_2.png)
+
+### SQLite
+
+1. After logging into your server via SSH, run the following to import your database:
+
+```
+sqlite3 database_name < /PATH/TO/DUMP.SQL 
+```
+
+2. Now login to your database and check to see if your tables exist by running the following:
+
+```
+sqlite3 database_name.db
+```
+
+3. Once you have entered SQLite, you will be greeted with a simlar message to the following:
+
+[![ssh sqlitegreeting](/assets/images/1.8/database-restore/ssh_sqlite_1.png)](/assets/images/1.8/database-restore/ssh_sqlite_1.png)
+
+3. Proceed to run the following:
+
+``` 
+.tables
+```
+
+4. Your SSH window should now look something like this:
+
+[![ssh sqlitetables](/assets/images/1.8/database-restore/ssh_sqlite_2.png)](/assets/images/1.8/database-restore/ssh_sqlite_2.png)
+
+
+### PostgreSQL
+
+1. After logging into your server via SSH, you will now need to enter PostgreSQL. Your method may slightly differ when logging in. Run the following to login:
+
+``` 
+su - postgres
+```
+
+[![ssh postgres](/assets/images/1.8/database-restore/ssh_postgresql_1.png)](/assets/images/1.8/database-restore/ssh_postgresql_1.png)
+
+2. To restore your database, you will need to run the following:
+
+``` 
+psql database_name < /PATH/TO/DUMP.SQL
+```
+
+3. You can now proceed to run the following to output your tables:
+
+``` 
+psql database_name
+```
+
+4. Followed by:
+
+``` 
+\dt
+```
+
+[![ssh postgrestables](/assets/images/1.8/database-restore/ssh_postgresql_2.png)](/assets/images/1.8/database-restore/ssh_postgresql_2.png)
+
+## Optional
+
+1. Now that you have restored your database, we are not quite done yet. You may now need to configure your database settings (.../inc/config.php).
```

Comment:
Some extra details on what might need to be changed here would be nice, along with a comment showing the interesting parts of the file (eg: the database configuration). Would be nice to quote the path, like `./inc/config.php` too (and notice the single `.`, rather than `...`).

Reply:
Possibly, but as this is only database restoring this page assumes that a forum is already in-place and we're only restoring a database, not creating a new forum with an old database. There's a slight difference IMO. If this is the case and we're only restoring a database, only the ```table_prefix``` would need to be set as everything else would be set via the original install.

Reply:
True, and for a straight restore you'd probably not need to change the prefix either.